src/index.d.ts

```diff
@@ -1,14 +1,32 @@
 declare module "use-bus" {
-  export interface EventAction {
-    type: string;
+  export interface EventAction<T extends string = string> {
+    type: T;
     [key: string]: any;
```

Comment:
What do you think about adding a second optional generic type for the value of `[key: string]` instead of `any` and default that new generic type to `any`